From 3b6b45477fff9a9b1dc8c8198a4241fccf06efc5 Mon Sep 17 00:00:00 2001
From: Ivan Pivovarov <ivan.pivovarov@lge.com>
Date: Mon, 12 Sep 2022 18:51:09 +0300
Subject: [PATCH] chrono namespace changed

:Release Notes:

:Detailed Notes:
Use different namespace in chrono instead of gcc-specific.

:Testing Performed:
bitbake media-codec-interface-clang
bitbake g-media-pipeline-clang

:QA Notes:
N/A

:Issues Addressed:
[WRP-6187] CCC: ose: Make clang a default option to build the webruntime
           and wam
[WRO-11443] Integrate meta-lg-webos-clang into meta-lg-webos
[WRP-310] Apply .patch files for hot-fix to corresponding repositories
[WRP-11630] Add Upstream-Status to existing .patch files in meta-lg-webos
[WRP-11864] CCC: Various build fixes and .patch files cleanup

Change-Id: I676dacd31ad5a70314ed97d607248adcdcdf7c62
---
Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/media-codec-interface/+/349078 chrono namespace changed]

 src/base/codec_types.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/base/codec_types.h b/src/base/codec_types.h
index 54b209f..ca54325 100644
--- a/src/base/codec_types.h
+++ b/src/base/codec_types.h
@@ -218,7 +218,7 @@ std::string FourccToString(uint32_t fourcc);
 VideoCodec VideoCodecProfileToVideoCodec(VideoCodecProfile profile);
 
 typedef std::vector<SupportedProfile> SupportedProfiles;
-typedef std::chrono::_V2::system_clock::time_point ChronoTime;
+typedef std::chrono::system_clock::time_point ChronoTime;
 
 }  //  namespace mcil
 
